lib/utils/color.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+abstract class ColorUtils {
```

Comment:
nit: drop the abstract class, it's not required. Just make the method a top level function.

Reply:
Pulled function into top-level :)